rustic-doc.el

```diff
@@ -311,9 +331,11 @@ NO-DL is primarily used for development of the filters."
                              (lambda (_p)
                                (message "Finished converting docs for std"))
                              "std")
+  (sleep-for 3)
   (rustic-doc-convert-current-package))
 
 (defun rustic-doc--start-process (name program finish-func &rest program-args)
+  (print (format "rustic-doc--start-process. name: %s program:  %s" name program))
```

Comment:
I think `message` is the common way to inform the user about progress.

Reply:
Also a relic from debugging. I should start taggin these as todo or something to remind myself to remove them.

Reply:
No problem, that's what reviews are for.